lib/client/index.js

```diff
@@ -1,15 +1,17 @@
-var mediatorSubscribers = require('./mediator-subscribers');
 var FileClient = require('./file-client');
 
 /**
  * Initialisation of the file module.
- * @param {Mediator} mediator
  * @param {object}  config -  module configuration
  * @param {object}  $fh - feedhenry sdk
  */
-module.exports = function(mediator, config, $fh) {
-
+module.exports = function(config, $fh) {
   //Initialising the subscribers to topics that the module is interested in.
   var fileClient = FileClient(config, $fh);
-  return mediatorSubscribers.init(mediator, fileClient);
+  
+  // OPTION 1 initialize global directly in module
```

Comment:
Hmm, I think this would make testing a bit more difficult, having to override the window.modrain for the tests.

I think passing it in is a cleaner option.